core/include/usModuleSettings.h

```diff
@@ -31,6 +31,8 @@
 
```

Comment:
We should use framework properties instead (can be done at some point later in time). Don't worry about the autoload stuff.